buildSrc/build.gradle

```diff
@@ -22,7 +22,7 @@ dependencies {
     // These are not directly included, they are just constrained if they are pulled in as
     // transitive dependencies.
     constraints {
```

Comment:
I think we might just be able to remove the constraint on org.json:json. It looks like this constraint was added to work around a security vulnerability, but our other libraries may now be sufficiently up-to-date so that we wouldn't attempt to use an older version.

I think we could remove the constraint and then generate lockfiles locally to see what version get.

This was added in https://github.com/DataBiosphere/terra-billing-profile-manager/pull/279.

Reply:
Hmmm, having trouble with getting the lockfiles to generate. But looking some at the dependency tree, it would appear that removing these will put us at an older version… I don't know if it is the older version that was flagged or not.

Reply:
Update: these constraints should be able to be removed once this PR merges, and we update our Sam clients:

https://github.com/broadinstitute/sam/pull/1348/files#diff-2279092b8f11cd53cb5d95128a978059f6c056c40fc2013f91caed84246228e5

Reply:
If you wait just a little bit, there will be a new Sam Client which fixes this vulnerability shortly! https://github.com/broadinstitute/sam/pull/1348

Reply:
https://broadinstitute.jfrog.io/broadinstitute/libs-release-local;build.timestamp=1707863807/org/broadinstitute/dsde/workbench/sam-client_2.13/0.1-6a84247/sam-client_2.13-0.1-6a84247.jar

Reply:
Caught up over Slack -- https://github.com/DataBiosphere/terra-billing-profile-manager/pull/404 is ready for review.  This PR should close out when it's merged.